miniscope_io/models/stream.py

```diff
@@ -35,6 +35,16 @@ class ADCScaling(MiniscopeConfig):
         11.3,
         description="Voltage divider factor for the Vin voltage",
     )
+    battery_max_voltage: float = Field(
+        10.0,
+        description="Maximum voltage of the battery."
+        "Scaled battery voltage will be 0 if it is greater than this value",
```

Comment:
maybe use -1 to make it unambiguous from the value actually being zero (which should never happen, but thinking about downstream data analysis)